src/core/Toast/Toast.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Toast } from './Toast';
+import { axeTest } from '../../utils/test';
+
```

Comment:
Additional tests are needed as all props in public API should be tested somehow.

Reply:
fixed All props tested

Reply:
Tests look good now in my opinion. One thing we could still add is testing forward refs but that might be an overkill for this component. Will resolve the conversation.